src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -465,8 +478,23 @@ private function loadCacheTagging(ContainerBuilder $container, XmlFileLoader $lo
         }
 
         $container->setParameter($this->getAlias().'.compiler_pass.tag_annotations', true);
-        $container->setParameter($this->getAlias().'.tag_handler.response_header', $config['response_header']);
+
         $container->setParameter($this->getAlias().'.tag_handler.strict', $config['strict']);
+
+        // If no response_header parameter has been already been set, use the one from the config
+        if (!$container->hasParameter($this->getAlias().'.tag_handler.response_header')) {
```

Comment:
this feels a bit magic. wdyt of instead having a validate step that sets the right response_header and separator? i think we should only change the headers if varnish is the only or the `default` client, and otherwise use the default Varnish::DEFAULT_HTTP_HEADER_CACHE_TAGS if the response_header is not set